### PR TITLE
No scl if not present

### PR DESCRIPTION
**Pull request by joepvd**

### CODE REVIEW

File: scheduled-jobs/scanning/unresolved-art-threads/Jenkinsfile [hunk 1]
```diff
@@ -16,7 +16,13 @@ node() {
                             string(credentialsId: "art-bot-slack-signing-secret", variable: "SLACK_SIGNING_SECRET")
                         ]) {
             retry(10) {
-                commonlib.shell(script: "scl enable rh-python38 -- scheduled-jobs/scanning/unresolved-art-threads/unresolved-thread-notification.py")
+                commonlib.shell(script: """
+                    if [[ -f /bin/scl ]]; then
+                        scl enable rh-python38 -- scheduled-jobs/scanning/unresolved-art-threads/unresolved-thread-notification.py
+                    else
+                        python3.9 -- scheduled-jobs/scanning/unresolved-art-threads/unresolved-thread-notification.py
```
Comment (locriandev) [inline scheduled-jobs/scanning/unresolved-art-threads/Jenkinsfile:23]: Why pinning 3.9? We might bump to 3.11...

**Review by locriandev — CHANGES_REQUESTED** [review]

**Review by locriandev — COMMENTED** [review]
/lgtm

File: scheduled-jobs/scanning/unresolved-art-threads/Jenkinsfile (hunk 1 above)
Comment (joepvd) [reply scheduled-jobs/scanning/unresolved-art-threads/Jenkinsfile:23]:
```suggestion
                        python -- scheduled-jobs/scanning/unresolved-art-threads/unresolved-thread-notification.py
```

**Pull request merged**